src/session.ts

```diff
@@ -3,6 +3,11 @@ import type { BigNumber } from 'ethers';
 import type { TransactionReceipt, TransactionResponse } from '@ethersproject/providers';
 import { Config, getConfig } from "@app/config";
 import { Unreachable, assert, assertEq } from "@app/error";
+import WalletConnect from "@walletconnect/client";
+import * as ethers from "ethers";
+import ModalWalletQRCode from "@app/Components/Modal/QRCode.svelte";
```

Comment:
We don't want to be importing UI components in a non-ui typescript module. These should be separate. This shouldn't be needed once you switch to our existing modal system anyway.

Reply:
Yeah. noted